lib/register.dart

```diff
@@ -0,0 +1,306 @@
+import 'package:InstiComplaints/modals.dart';
+import 'package:flutter/material.dart';
+import 'dart:math';
+
+class BackgroundMaker extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: Stack(
+          fit: StackFit.expand,
+          children: <Widget>[
+            Container(
+              constraints: BoxConstraints.expand(),
+              color: Colors.white,
+              child: RegisterPage(),
+            ),
+            ClipPath(
```

Comment:
The ClipPath widget takes the CurveClipper we made to clip the path as clipper and we add a Container child widget to it with the desired colour to paint our needed Clip.